### PR TITLE
[BIP-119] Slim down motivation, add more references

### PR DESCRIPTION
**Pull request by JeremyRubin**
The level of detail in the motivations for BIP-119 was excessive, this PR removes the lengthy motivation section and adds more links out to various resources.

### CODE REVIEW

File: bip-0119.mediawiki [hunk 1]
```diff
@@ -39,125 +39,24 @@ The recommended standardness rules additionally:
...
+This is a general definition based on the legal definition which even simple
+scripts using CSV would satisfy.  Covenants in Bitcoin transactions usually
+refer to restrictions on where coins can be transferred.  Covenants can be
+useful to construct smart contracts. As covenants are complex to implement and
+risk of introducing fungibility discriminants they have not been seriously
+considered for inclusion in Bitcoin.
+
+This BIP introduces a simple covenant called a *template* which enables a
+limited set of highly valuable use cases without significant risk. BIP-119
+templates allow for non-recursive fully-enumerated covenants with no dynamic
```
Comment (1440000bytes) [inline bip-0119.mediawiki:52]:
```suggestion
templates allow for '''non-recursive''' fully-enumerated covenants with no dynamic
```

File: bip-0119.mediawiki [hunk 2]
```diff
@@ -39,125 +39,24 @@ The recommended standardness rules additionally:
...
+state. CTV serves as a replacement for a pre-signed transaction oracle, which
+eliminates the trust and interactivity requirements.  Examples of uses include
+wallet vaults, non-interactive payment channel creation, congestion controlled
```
Comment (1440000bytes) [inline bip-0119.mediawiki:55]:
```suggestion
vaults, non-interactive payment channel creation, congestion controlled
```

**Review by 1440000bytes — APPROVED** [review]
ACK https://github.com/bitcoin/bips/pull/1320/commits/de0ff362fc70a814978b6b11ee8887109a6bdc40

This would make it easier for developers to review the BIP and not get confused with the use cases. 2 suggestions that can be ignored.

File: bip-0119.mediawiki [hunk 3]
```diff
@@ -39,125 +39,24 @@ The recommended standardness rules additionally:
...
-doesn't need to know the totality of the outputs committed to by that output,
-they only have to verify their own sub-tree will pay them. These trees can
-then, using a top-level Schnorr key, be interactively updated on a rolling basis
-forming a "Payment Pool".
+Covenants are restrictions on how a coin may be spent beyond key ownership.
+This is a general definition based on the legal definition which even simple
+scripts using CSV would satisfy.  Covenants in Bitcoin transactions usually
+refer to restrictions on where coins can be transferred.  Covenants can be
+useful to construct smart contracts. As covenants are complex to implement and
+risk of introducing fungibility discriminants they have not been seriously
```
Comment (bucko13) [inline bip-0119.mediawiki:47]: should this be "risk introducing fungibility discriminants"?

comma after "discriminants" to split up the clauses.

**Review by bucko13 — COMMENTED** [review]
ACK. Just a couple of small copy edit nits.

**Comment by AdamISZ** [comment]
ACK for this simplification, the very large detailing of application constructs seemed not appropriate for this type of doc.

Having said that, it may be overkill to give only names and leave everything else to citations; short summaries in-text of a few major applications are common (off the top of my head I remember BIP65, BIP340, I'm sure there are many, many others!). But I mean, this works, as is.

(Another thing to consider is ordering: I never liked having to scroll so far down to find the Rationale section).

**Comment by JeremyRubin** [comment]
@AdamISZ I think the BIP-65 example makes me think that it may be best to err on the lighter side w.r.t. applications...

None of the given examples are particularly in common use, if at all? Where they have been used is in HTLCs, which are (IMO -- I could be misreading) very different from what is stated in the BIP.

Similarly, BIP-340 links to MuSig, which has a flaw rendering it "less usable"/unprovable.

This leads me to believe that it's better to be abstract in the applications section, as previous examples in BIPs don't fully stand the test of time, but the concepts do.

**Comment by AdamISZ** [comment]
I think that's a pretty decent overall take fwiw.

(Re: BIP340's link to the 2018/068 paper, it's no longer the flawed version there, but the patched up one, so it's not technically a mistake, but yeah for sure once the MuSig2 BIP is actually assigned they will presumably want to update it since that's significantly preferable).

**Comment by luke-jr** [comment]
IMO better to keep it all in the BIP (links die), but you're the author...

**Pull request merged**